x/concentrated-liquidity/lp_test.go

```diff
@@ -551,6 +551,95 @@ func (s *KeeperTestSuite) TestIsInitialPosition() {
 	}
 }
 
+func (s *KeeperTestSuite) TestUpdatePosition() {
+	type updatePositionTest struct {
+		poolId          uint64
+		owner           sdk.AccAddress
+		lowerTick       int64
+		upperTick       int64
+		liquidityDelta  sdk.Dec
+		amount0Expected sdk.Int
+		amount1Expected sdk.Int
+		expectedError   bool
+	}
+
+	tests := map[string]updatePositionTest{
+		"happy path": {
```

Comment:
We should have more ttest cases such as:
- updating an existing position
- trying to call update position on a non-existent position due to ticks being different
- success case with a positive liquidity delta
- success case with a negative liquidity delta

Reply:
test case for `updating an existing position` is the first test case if I understood you correctly

Reply:
same with positive liquidity delta?

Reply:
added other test cases